### PR TITLE
Add: Blog post about timekeeping in OpenTTD 14

### PR DESCRIPTION
**Pull request by 2TallTyler**
As promised, my dev diary about timekeeping in OpenTTD 14.

### To do

- [x] Come up with a better title
- [x] Add images
- [x] Create Steam image
- [ ] Write text for Socials announcements

### CODE REVIEW

File: _posts/2024-02-24-timekeeping [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+Calendar units are the classic OpenTTD experience, with no changes.
+Wallclock units use minutes anywhere months are normally displayed, with 12-minute periods substituting for years. Timetables are expressed in seconds.
+
+[Image of some GUIs with wallclock unit stats]
+
+Where a date would normally be displayed (timetable start date, subsidy expiration date, etc.), the game shows an offset in seconds or minutes from the current moment.
+
+[Image of the timetable GUI showing seconds in the arrival/departure pane]
+
```
Comment (zephyris) [inline _posts/2024-02-24-timekeeping:73]: I think you need somewhere, possibly here, a super simple tutorial about how to set the settings for different play styles. Eg. If you want time to stay the same as it always was, set time to "Calendar" mode here or here. If you want a month to last 5 minutes, then you need to set time to "Wallclock" mode and minutes per year to 12 * 5 = 60 here. If you want to stay in the same year forever, set the minutes per year to "0 (frozen time)" here. A screenshot of the settings to make this super-clear would be useful too.

File: _posts/2024-02-24-timekeeping [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+The ugliest solution is to display two dates.
+But OpenTTD's user interface is confusing enough.
+Adding a second date is not a good user experience.
+
+Nielsm's proposal capitalizes on the fact that a day in OpenTTD is about two seconds, meaning that a 30-day month is about a minute.
+We can simply make this a bit more precise, then display economy time in real seconds and minutes!
+
+The first step was making the seconds-per-day a bit more precise.
+The smallest unit of time in OpenTTD is a tick, of which there are 74 in a standard day.
+Prior to OpenTTD 14, one tick lasted 27 milliseconds, making a day 2.22 seconds.
```
Comment (stormcone) [inline _posts/2024-02-24-timekeeping:57]:
```suggestion
Prior to OpenTTD 14, one tick lasted 30 milliseconds, making a day 2.22 seconds.
```

File: _posts/2024-02-24-timekeeping (hunk 1 above)
Comment (michicc) [reply _posts/2024-02-24-timekeeping:73]: I have to agree here. There's nothing wrong with talking about the development/behind-the-scenes, but I do agree a section with some "how to use it" would be good.

**Comment by michicc** [comment]
General side note: We usually do one sentence per line for markdown, but that is just for review of course as the formatting doesn't care. Whether this is an imortant point, eh 🤷

File: _posts/2024-02-24-timekeeping [hunk 3]
```diff
@@ -0,0 +1,143 @@
+---
```
Comment (TrueBrain) [inline _posts/2024-02-24-timekeeping:1]: please make sure your posts end with `.md`, otherwise Jekyll doesn't know how to render it :)

**Comment by 2TallTyler** [comment]
Proposed Steam image:

timekeeping-steam

[timekeeping-steam.zip](https://github.com/OpenTTD/website/files/14726613/timekeeping-steam.zip)

**Review by michicc — APPROVED** [review]

**Pull request merged**